mcs/class/corlib/Test/System.Threading/ReaderWriterLockTest.cs

```diff
@@ -94,6 +94,7 @@ private void IsWriterLockHeld_2 ()
 		}
 				
 		[Test]
+		[Category ("MultiThreaded")]
```

Comment:
I don't see this category being disabled anywhere in sdks/ ?

Reply:
Yeah, that's missing from this PR, I got some fixing to do here anyways.